Userland/Libraries/LibChess/Chess.cpp

```diff
@@ -221,42 +222,116 @@ String Move::to_algebraic() const
 
 Board::Board()
 {
-    // Fill empty spaces.
-    for (int rank = 2; rank < 6; ++rank) {
-        for (int file = 0; file < 8; ++file) {
+    MUST(load_fen("rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1"));
```

Comment:
Please add a short comment explaining what this does for those unfamiliar with this format (default position of all the pieces, I assume) :)